src/components/send/components/TXStatus.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Translate } from 'react-localize-redux'
+
+const StyledContainer = styled.div`
+    display: flex;
+    align-items: center;
+`
+const Indicator = styled.span`
+    display: inline-block;
+    width: 9px;
+    height: 9px;   
+    background-color: ${props => props.color};
+    border-radius: 50%;
+    margin-right: 10px;
+`
+
+const TXStatus = ({ status }) => {
+
+    const getStatusColor = () => {
```

Comment:
Would prefer if this was a pure method that lived in module scope above our component definition, since it doesn't need to be re-created every time we render; we just need to make it take `status` as an argument instead of snorkeling it from the parent closure